__tests__/components/Header.test.tsx

```diff
@@ -1,50 +1,43 @@
 import { render, screen } from '@testing-library/react'
 import '@testing-library/jest-dom/extend-expect'
 import { axe, toHaveNoViolations } from 'jest-axe'
-import Header from '../../components/Header'
-
 import { useRouter } from 'next/router'
-
-// mocks useRouter to be able to use component' router.asPath
-jest.mock('next/router', () => ({
-  useRouter: jest.fn(),
-}))
-
-// the code below is to avoid the following error:
-//    "An update to Link inside a test was not wrapped in act(...)"
-jest.mock('next/link', () => ({
-  __esModule: true,
-  default: ({ children, href }) => (
-    <children.type {...children.props} href={href} />
-  ),
-}))
+import Header from '../../components/Header'
 
 const defaultRouterObj = {
   pathname: '/',
   asPath: '/',
   locale: 'en',
 }
 
+// mocks useRouter to be able to use component' router.asPath
+jest.mock('next/router', () => ({
+  useRouter: jest.fn(() => defaultRouterObj),
+}))
+
 expect.extend(toHaveNoViolations)
 
 describe('Header', () => {
-  useRouter.mockImplementation(() => defaultRouterObj)
-
   it('renders Header in English', () => {
-    render(<Header />)
+    render(<Header gocLink="testGocLink" skipToMainText="testSkipToMainText" />)
     const HeaderLang = screen.getByText('Français')
     expect(HeaderLang).toBeInTheDocument()
   })
 
   it('renders Header in French', () => {
-    useRouter.mockImplementation(() => ({ ...defaultRouterObj, locale: 'fr' }))
-    render(<Header />)
+    ;(useRouter as jest.Mock).mockImplementationOnce(() => ({
```

Comment:
Stray semicolon here?

Reply:
Yeah, it's added when I save. I believe Prettier is adding it.
https://prettier.io/docs/en/rationale.html#semicolons

Reply:
https://standardjs.com/rules.html#semicolons

Reply:
I can change it like this if you prefer:
```tsx
const mockUseRoutrer = (useRouter as jest.Mock)
mockUseRoutrer.mockImplementationOnce(...)
```

Reply:
Change applied. TIL semicolon usage with no-semicolon javascript. hehe

Reply:
TIL as well haha